src/services/placeService.js

```diff
@@ -17,7 +18,7 @@ class PlacesService {
 
       return response.data;
     } catch (e) {
-      console.log(e.message);
+      notificationsStore.addNotification({ message: 'Error fetching place data' + e.message });
```

Comment:
Please explicitly provide type for notification

Reply:
I don't think that we need to do this for an error as it's default type.
Most of notifications that we create will be an error-notification, why do we need to duplicate type in each of them?

Reply:
And addNotification placed in `catch` block, as for me it seems explicit that it is error notification

Reply:
This is faster not to show, but for developer who will make changes here it will be additional steps to understand where type comes from - and in this case it is hidden inside default condition of switch case block